test/AISmart.Application.Tests/GAgent/TwitterGAgentTest.cs

```diff
@@ -24,21 +24,12 @@ public TwitterGAgentTest(ITestOutputHelper output)
     //https://developer.twitter.com/apitools/api
 
     
-    [Fact]
-    public async Task PostTwittersAsyncTest ()
-    {
-        var accessToken = "";
-        var accessTokenSecret = "";
-        var resp = await _twitterProvider.PostTwitterAsync( "Today is Friday！", accessToken, accessTokenSecret);
```

Comment:
we don't need to unit test `PostTwitterAsync`?

Reply:
我这里之前是用来集成测试的，unit test理论上应该写一个mock的twitter，我需要再实现一下，然后再提交

Reply:
Ok please implement the corresponding unit test. I will approve this PR first so that everyone else who is using dev is not blocked by the compile error. Thank you.

Reply:
sure